build/config/RepoUtilData.json

```diff
@@ -41,7 +41,8 @@
                 "xunit",
                 "RoslynTools.*",
                 "^Microsoft.?Build.*",
-                "GitLink"
+                "GitLink",
+                "Microsoft.NETCore.Runtime.CoreCLR"
```

Comment:
This is makeing sure that the MicrosoftNETCoreRuntimeCoreCLRVersion element gets generated to Dependencies.props